services/core-web/src/components/Forms/permits/conditions/SubCondition.js

```diff
@@ -76,12 +78,41 @@ const SubCondition = (props) => {
             </Col>
           </Row>
         </Col>
-        <Col span={3} className="float-right">
+        <Col span={4} className="float-right">
           {!isEditing && !props.isViewOnly && (
-            <div>
+            <div className="float-right">
               <AuthorizationWrapper permission={Permission.ADMIN}>
                 <Button
                   ghost
+                  className="no-margin"
+                  size="small"
+                  type="primary"
+                  onClick={() => {
+                    props.reorderConditions(props.condition, true);
+                  }}
+                  disabled={props.editingConditionFlag}
+                >
+                  <Icon type="up" theme="outlined" />
+                </Button>
+              </AuthorizationWrapper>
+              <AuthorizationWrapper permission={Permission.ADMIN}>
```

Comment:
This isn't specific to this PR, but curious what permissions should be on these actions. 

I think edit_permits would be better than admin. I'll check with Brian/Aaron